src/Microsoft.TestPlatform.CoreUtilities/Utilities/UILanguageOverride.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.VisualStudio.TestPlatform.CoreUtilities.Utilities
+{
+    public static class UILanguageOverride
```

Comment:
Frankly I would rather see this shared via a shared project, as the code is duplicated among projects but they don't have any dependency on each other, and this just adds a possible point of failure when the object model mismatches.

@Haplois @jakubch1  @pavelhorak thoughts?